internal/mithril-metric/Makefile

```
@@ -0,0 +1,19 @@
+.PHONY: all build test check doc
+
+CARGO = cargo
+
+all: test build
+
+build:
+	${CARGO} build --release
+
+test:
+	${CARGO} test
+
+check:
+	${CARGO} check --release --all-features --all-targets
+	${CARGO} clippy --release --all-features --all-targets
+	${CARGO} fmt --check
+
+doc:
+	${CARGO} doc --no-deps --open --features full
```

There's no feature on this new crate :).
```suggestion
	${CARGO} doc --no-deps --open
```